client/blocks/login/login-form.jsx

```diff
@@ -368,30 +367,6 @@ export class LoginForm extends Component {
 						</div>
 					</div>
 
-					{ config.isEnabled( 'signup/social' ) && (
-						<p className="login__form-terms">
-							{ preventWidows(
-								this.props.translate(
-									// To make any changes to this copy please speak to the legal team
-									'By creating an account, ' +
-										'you agree to our {{tosLink}}Terms of Service{{/tosLink}}.',
-									{
-										components: {
-											tosLink: (
-												<ExternalLink
-													href={ localizeUrl( 'https://wordpress.com/tos/' ) }
-													target="_blank"
-													rel="noopener noreferrer"
-												/>
-											),
-										},
-									}
-								),
-								5
-							) }
-						</p>
-					) }
-
```

Comment:
Should this be removed?

Reply:
I meant to add a review comment explaining this -- thanks for catching it!

This text is present on the sign-up pages already. This was accidentally added to the (unreleased) WooCommerce Jetpack login flow as well, when I was building the Muriel login inputs. So this is just fixing that mistake.